client/src/components/GoogleSignIn.jsx

```
@@ -31,28 +36,62 @@ const GoogleSignIn = () => {
+      {message && (
+        <Alert severity="error" sx={{ width: '100%', marginBottom: 2 }}>
+          {message}
+        </Alert>
+      )}
+
+      <button
+        onClick={handleGoogle}
+        className="cursor-pointer flex items-center justify-center w-full bg-white border border-gray-300 text-gray-700 p-2 rounded-full shadow-md hover:bg-gray-100"
+      >
+        {loading ? (
+          <div className="w-5 h-5 border-2 border-gray-200 border-t-gray-600 rounded-full animate-spin"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)